src/workflows/demo_pipeline/config.vsh.yaml

```diff
@@ -0,0 +1,52 @@
+functionality:
+  name: demo_pipeline
+  namespace: workflows
+  description: |
+    An example pipeline and project template.
+
+  arguments:
+    - name: "--input"
+      alternatives: [ "-i" ]
+      type: file
+      required: true
+      multiple: true
+      multiple_sep: ";"
+      description: fastq files
+      example: file1.tar.gz;file2.tar.gz 
+    - name: "--output"
+      alternatives: [ "-o" ]
+      type: file
+      direction: output
+      required: true
+      description: Output directory to write reports to.
+      example: qc/
+
+  resources:
+    - type: nextflow_script
+      path: main.nf
+      entrypoint: run_wf
+
+  dependencies: 
+    - name: demo/combine_columns
+      repository: local
+    - name: demo/remove_comments
+      repository: local
+    - name: demo/take_column
+      repository: local
+    - name: split/expand
```

Comment:
`expand_glob`? I used to have `transpose`, but that's a bit too generic.

Suggestions are welcome!

Reply:
It's possible to specify the input as `file1.tsv;file2.tsv,...` as well. That's not really a glob either.

As a matter of fact, I'm starting to think this could be the default behavior of a Viash module: convert `Channel[List]` into multiple events.

Reply:
`vsh_flatten`, because it's effectively a `flatMap` on `state.input`?